configs/dla/cascade_mask_rcnn_dla_60_fpn_20e_coco.py

```diff
@@ -0,0 +1,18 @@
+_base_ = '../cascade_rcnn/cascade_mask_rcnn_r50_fpn_20e_coco.py'
+
+model = dict(
+    pretrained='open-mmlab://dla60',
```

Comment:
Is this from a third-party library?

Reply:
we need to upload model checkpoints if necessary.

Reply:
> Is this from a third-party library?
Yes, but I did some modification to match the mmdet-style codes. [Here](https://github.com/ucbdrive/dla) is the official implement

Reply:
Could I get the pretrained weight using 'open-mmlab://dla60' now?